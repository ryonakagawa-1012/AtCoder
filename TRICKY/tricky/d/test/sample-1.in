2
3 2
Takahashi is the 2nd place
Takahashi overtakes Aoki
HechoSamurai overtakes Aoki
2 1
Takahashi is the 1st place
Takahashi overtakes Aoki
