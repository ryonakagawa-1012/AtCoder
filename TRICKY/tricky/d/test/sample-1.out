Takahashi might get the 1st place
The memo must be wrong
